Prace_domowe/Praca_domowa3/Grupa4/Marcin Wilk/PD3.ipynb

```diff
@@ -0,0 +1,1085 @@
+{
```

Comment:
<p>Czemu uważasz, że bayes poradził sobie lepiej ze zmiennymi jakościowymi? </p><p>Serio, to jest dla mnie nowość i chętnie się dowiem \:)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>Bayes estymuje pstwa zmiennych jakościowych po prostu jako frakcje w rozkładzie wielomianowym, nie zakłada na nich rozkładu ciągłego takiego jak normalny ani nie robi nieparametrycznych estymacji gęstości, więc dla niego to czy zmienne zostaną zakodowane jako 1,2,3,4 czy 1,2,3,789 nie robi żadnej różnicy, a w regresji jednak jesteśmy bardziej zależni od skali zmiennych jakościowych</p>

Reply:
No tak, ale tu te zmienne mają naturalną kolejność i skalę - stopień zachmurzenia 0-8. 
Faktycznie NB ma najlepszy recall, ale czy to jasno wynika z powodu lepszego działania na zmiennych dyskretnych? Kategoryczne to one nie są